src/snowflake/snowpark/modin/plugin/PANDAS_CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 - Added partial support for `SeriesGroupBy.apply` (where the `SeriesGrouBy` is obtained through `DataFrameGroupBy.__getitem__`).
 - Added support for `pd.NamedAgg` in `DataFrameGroupBy.agg` and `SeriesGroupBy.agg`.
 - Added support for `Series.str.slice`.
+- Added support for `DataFrame.pivot_table` with no `index` parameter.
```

Comment:
Can you mention `margins` here too?

Reply:
We already support margins - this PR adds support for no index parameter, but additional support had to be added for no index + margins - does that still warrant adding a note about margins specifically? I'm happy to, but I feel as though users would expect that margin would work with no index after this considering that we already support the margins parameter.

Reply:
Yup - noticed that that was the case, and went ahead and updated the first one. I'll update the second, and since we are changing it, i'll add that margins is supported now.